examples/cpp/battleship/protocol.cpp

```diff
@@ -30,7 +30,7 @@ RoundMessage::Content RoundMessage::decode() const {
 }
 
 InitMessage Battleship::init() {
-  risc0::MethodId methodId = risc0::makeMethodId("examples/cpp/battleship/init_method");
+  risc0::MethodId methodId = risc0::MethodId::fromElfFile("examples/cpp/battleship/init_method");
```

Comment:
These actually should be a `loadMethodId` since bazel already spent the time to compute `.id` files and left them in runfiles.

See: #167